### PR TITLE
Added --convert-only option and another test

### PR DESCRIPTION
**Pull request by nippoo**
The --convert-only option allows users to generate .high.kwd, .low.kwd, and .raw.kwd files using klusta without spike detection. A single chunk is detected, but the rest is ignored. This takes much less time than running the full spike detection algorithm every time.

### CODE REVIEW

**Comment by rossant** [comment]
What is the difference with --debug?

**Comment by nippoo** [comment]
--debug only processes the first chunk. --convert-only converts the whole file, also performing spike detection for the first chunk because otherwise KlustaViewa/TraceView will refuse to open (this is another bug which we probably don't need to fix before phy).

So this is useful if, for example, you don't want to save a .raw.kwd but then (after clustering) decide you want to use TraceView and look at the spikes or whatever - will convert files in a few minutes rather than a few hours.

Basically this saves a lot of time if you don't want to do spike detection all over again but do want to convert from .dat/.ns5/.bin to .kwd.

**Comment by rossant** [comment]
OK

BTW @kdharris101 I can't remember what was the last decision regarding .dat/.kwd. How will we store raw data in the next version of the file format (presumable NWB)? With flat binary files or HDF5 files?

**Comment by kdharris101** [comment]
The raw data format has not been decided, but will be polymorphic: it could be encoded in a HDF5 file (this is what Allen Inst will probably do) or in native format (what Svoboda lab will do).
